pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -469,18 +479,10 @@ func (r *ReconcileCnsRegisterVolume) Reconcile(ctx context.Context,
 	if isBound {
 		log.Infof("PVC: %s is bound", instance.Spec.PvcName)
 		if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.PodVMOnStretchedSupervisor) {
-			log.Info("Loading CnsVolumeInfo Service to persist mapping for VolumeID to storage policy info")
-			volumeInfoService, err := cnsvolumeinfo.InitVolumeInfoService(ctx)
-			if err != nil {
-				log.Errorf("Error initializing volumeInfoService. Error: %+v", err)
-				return reconcile.Result{RequeueAfter: timeout}, nil
-			}
-			if volumeInfoService != nil {
-				log.Infof("Successfully initialized VolumeInfoService")
-			}
 			// Create CNSVolumeInfo CR for static pv
-			err = volumeInfoService.CreateVolumeInfoWithPolicyInfo(ctx, volumeID, instance.Namespace,
-				volume.StoragePolicyId, storageClassName, vc.Config.Host)
+			capacity := resource.NewQuantity(capacityInMb, resource.BinarySI)
```

Comment:
I think we should convert  capacityInMb to size in Bytes.

Reply:
Done

Reply:
Done